### DISCUSSION TITLE
[Request] Pressing the sound button in the system tray opens the Win11 layout

### DISCUSSION THREAD

**Discussion by Archilyte** [valinet/ExplorerPatcher#2454] [Ideas]
I wanted to request this because the new layout is actually pretty cool now, it even has the right click menu built-in. It's accessed just like the WIn11 WI-FI layout

**Comment by y0sh1ne** [comment]
I agree it. And win11 layout provides a separate volume adjustment button for each app. Such as bellow:
![image](https://github.com/valinet/ExplorerPatcher/assets/150506715/36ca0709-4bbf-461c-93af-857fabccaf1b)
